src/state/ppangoChef/hooks.ts

```diff
@@ -436,3 +454,70 @@ export function useIsLockingPoolZero() {
 
   return pairs;
 }
+
+/**
+ * This hook returns the extra value provided by super farm extra reward tokens
+ * @param rewardTokens array os tokens
+ * @param rewardRate reward rate in png/s
+ * @param multipliers multipler fro each token provider in rewardTokens
+ * @param balance valueVariables from contract
+ * @param pairPrice pair price in wrapped gas coin
+ * @returns return the extra percentage of apr provided by super farm extra reward tokens
+ */
+export function useExtraFarmApr(
+  rewardTokens: Array<Token | null | undefined> | null | undefined,
+  rewardRate: BigNumber,
+  multipliers: JSBI[] | undefined,
+  balance: BigNumber,
+  pairPrice: Price,
+) {
+  const chainId = useChainId();
+  // remove png and null
+  const _rewardTokens =
+    rewardTokens
+      ?.map((token) => {
+        if (!token) {
+          return undefined;
+        }
+        return token;
+      })
+      ?.filter((token) => !!token && !PNG[chainId].equals(token)) || ([] as (Token | undefined)[]);
```

Comment:
we dont need extra map here. you can just use one filter loop and check condition